src/brian.py

```diff
@@ -287,6 +280,14 @@ def main():
     while True:
         time.sleep(1)
 
-if __name__ == "__main__":
-    main()
 
+if __name__ == "__main__":
+    from utils import log_config
+    log = log_config.log()
+    log.info(f"BRIAN BOOTED")
+    try:
+        main()
+        log.info(f"BRIAN EXITED")
+    except Exception as main_exception:
+        log.critical("BRIAN CRASHED", error=main_exception)
+        log.exception("BRIAN CRASHED", exception=main_exception)
```

Comment:
I throw both the CRITICAL and EXCEPTION because they are handled differently. The CRITICAL is logged at that level,  but the exception is not rendered in console with Rich. The EXCEPTION however does render it cleanly with Rich. This means we get a very well formatted Trace back when ever we crash in console.